cms/djangoapps/contentstore/views/public.py

```diff
@@ -66,6 +66,6 @@ def login_page(request):
 def howitworks(request):
     "Proxy view"
     if request.user.is_authenticated():
-        return redirect('/course/')
+        return redirect('/home/')
```

Comment:
@marcotuts @cahrens We do redirect to /home/, in https://github.com/open-craft/edx-platform/blob/content_libraries/11-UI-updates/cms/djangoapps/contentstore/views/course.py#L235